### ISSUE TITLE
[Bug]: Can't accept terms of usage via mobile app 

### ISSUE THREAD

**Issue by Adiutant** [MetaMask/metamask-mobile#6826]
### Describe the bug

Android 12.0.0 EMUI 12.0.0 Huawei p30 pro. It is impossible to accept the terms of use, the widget displaying them is white and without information. Unable to continue working

### Steps to reproduce

1. Download app from google play
2. Open app
3. Click "Start working"
4. Create new wallet

### Version

7.2.0

### Build type

None

### Browser

Other (please elaborate in the "Additional Context" section)

### Operating system

Other (please elaborate in the "Additional Context" section)

### Additional context

Android 12.0.0 EMUI 12.0.0 Huawei p30 pro.

**Comment by anaamolnar** [comment]
Hey @Adiutant. Thanks for reporting! Could you send a screen recording of this? Thank you!

**Comment by plasmacorral** [comment]
@Adiutant Sorry for the difficulty here, but thank you for taking the time to share this report with us.  I just experienced something similar today while I was recording my screen, but the pages were blocked in my recording.

@anaamolnar I experienced this on Android 13 with Pixel 5a using v7.5.0 build 1167.  With the production build, these screens are not captured in the screen recording for security.  In my experience today, the terms screen was blank for an extended period and I too was unable to proceed with app use.  After what felt like an unusually long time, the terms were eventually rendered and I could proceed. The slow presentation of terms _may_ be related to network connectivity. 

If this is something that we cannot render more promptly, the observation that I will share is that it would be incredibly helpful to display a spinner or something to help a user appreciate that more time is needed.
